lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -359,6 +358,27 @@ sub log_entry {
     my $get_text =
         sub { my $color = shift // 0; $text_data{$color} //= $self->format_line($data, {color => $color}) . "\n"; return $text_data{$color}; };
 
+    # remove substitution context from message
+    if ($data->{message})
+    {
+        $data->{message} =~ s/\".*//;
+    }  
+    # Prepare the JSON object with the required fields
+    my %log_data = (
+        message   => $data->{message},
+        severity  => $data->{severity},
+    );    
+    
+    if ($self->{context} && ref($self->{context}) eq 'HASH'){
+        my @keys = keys %{$self->{context}}; # Get the keys from the context hash
+ 
+        foreach my $key (@keys) {
+            $log_data{$key} = $self->{context}->{$key};
+        }
+        my $json_string = $JSON->encode(\%log_data);
```

Comment:
`use Log::Any::Adapter ('DERIV', stderr => 'json')` can make log items as a json string directly, why we need encode it again here ?

Reply:
the message is formulated when the context is auto added it looks like this '/correlation_id:"dasd"/, we are combining the context keys as well as part of message